client/src/reducers/undoableConfig.js

```diff
@@ -30,6 +31,9 @@ const skipOnActions = new Set([
   "writable obs annotations - save complete",
   "writable obs annotations - save started",
   "writable obs annotations - save error",
+  "autosave: genesets started",
+  "autosave: genesets error",
+  "autosave: genesets complete",
 
```

Comment:
It appears that the gene set UI-related actions are missing.  I believe they should *not* generate a new history stack element, so you should put them in the skip list.